agent/util-scripts/pbench-ara.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think we should create an `agent/ansible/` directory hierarchy to place playbooks and roles instead of using `util-scripts`.